### PR TITLE
Adds video link field to story model

### PR DESCRIPTION
**Pull request by s278gupt**
Part 1 of https://github.com/uwblueprint/shoe-project/issues/185

See https://github.com/uwblueprint/shoe-project/issues/185#issuecomment-765444247: `we need to add a new column to the model to store a youtube link (video)`

### CODE REVIEW

**Review by AbhiPrasad — COMMENTED** [review]
Mind also updating the TypeScript types found at https://github.com/uwblueprint/shoe-project/blob/main/ui/src/types/index.tsx#L11?

**Comment by nimwijetunga** [comment]
Also please add some random youtube links to the stories.json file so we can test it once the frontend is ready

**Comment by s278gupt** [comment]
@nimwijetunga feel free to merge this if approved

**Review by hanlinc27 — COMMENTED** [review]
In the future, we should remember to be adding the embedded youtube URL (found when you click share video) and is the form "youtube.com/embed/...." rather than the watch URL

File: data/stories.json [hunk 1]
```diff
@@ -1,431 +1,437 @@
...
+        "author": {
+            "first_name": "Tanaz",
+            "last_name": "Bhathena",
+            "origin_country": "India"
+        },
+        "video_url": "https://www.youtube.com/watch?v=ao6jHx27gB8",
```
Comment (nimwijetunga) [inline data/stories.json]: can you change this to the embed link (take the parameter v and place it in the url path)
i.e https://www.youtube.com/watch?v=ao6jHx27gB8 -> https://www.youtube.com/embed/ao6jHx27gB8

Comment (nimwijetunga) [reply data/stories.json]: Also please do this for the other youtube links

**Review by nimwijetunga — CHANGES_REQUESTED** [review]

File: data/stories.json (hunk 1 above)
Comment (AbhiPrasad) [reply data/stories.json]: Hmm should we validate for this? Like say the video URL must follow a certain regex or the POST is not accepted?

Comment (nimwijetunga) [reply data/stories.json]: We cant expect shoe project to do this b/c its kind of annoying to get this url. Instead given a regular youtube url we can convert it to the embed version by extracting the query parameter value for `v`. I think that should be a task for the story upload form (on the backend)

**Review by nimwijetunga — APPROVED** [review]

**Pull request merged**